projects/admin/src/app/routes/documents-route.ts

```diff
@@ -70,7 +73,7 @@ export class DocumentsRoute extends BaseRoute implements RouteInterface {
             aggregationsExpand: ['document_type'],
             aggregationsBucketSize: 10,
             itemHeaders: {
-              Accept: 'application/rero+json, application/json'
```

Comment:
Why did you changed this? `application/json` still exists and it is just a fall back.

Reply:
because, `application/json` break the display of informations.